pkg/cmd/cli/download/get.go

```diff
@@ -20,6 +21,7 @@ func NewGetCommand(f client.Factory, use string) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			logLevel := logLevelFlag.Parse()
 			logger := logging.DefaultLogger(logLevel, formatFlag.Parse())
+			logger.Debugf("setting log-level to %s", strings.ToUpper(logLevel.String()))
 			logger.Infof("The command, datamgr download get, is called")
```

Comment:
Let's change "The command, datamgr download get, is called " to "Datamgr download get called"